src/Observable.js

```diff
@@ -0,0 +1,69 @@
+var Disposable = require("src/Disposable");
+var Subscriber = require("src/Subscriber");
+
+module.exports = function() {
+    
+    function Observable(source) {
+        this.source = source;
+    }
+    
+    Observable.create = function(subscribe) {
+        return new Observable({ subscribe: subscribe });
+    };
+    
+    Observable.template = function(SubscriberType) {
+        function ObservableTemplate(source) {
```

Comment:
Yes, `ObservableTemplate` is a shortcut for creating each specific `Observable` subclass for each operator, e.g. `MapObservable`, `ZipObservable`, etc., since its job is really just to instantiate a specific `Generator` constructor with the appropriate args.